services/server/src/routes/index.ts

```diff
@@ -18,6 +19,7 @@ import cache from "@/routes/cache";
 import session from "@/routes/session";
 
 export default {
+  availabilitySlot,
```

Comment:
[Optional] Let's put it and the end. At the top you can see that all are functions but below you can see regular routes (similar to `availabilitySlot`)